packages/internal/src/node/createBundles.js

```diff
@@ -23,7 +23,7 @@ export const createBundlesFromAbsolute = async sourceBundles => {
     }
     const bundle = match[1];
 
-    const args = cacheToArgs.get(cache) || ['--to', cache];
+    const args = cacheToArgs.get(cache) || ['--cache-js', cache];
```

Comment:
What does this change do? is it compatible?

Reply:
This does not change behavior. When I added `--cache-json`, I also back-filled `--cache-js` to replace the less-specific `--to`. The usage states `bundle-source [--cache-json | --cache-js] <etc>`, so this just brings usage into alignment.